origin/tunnel.go

```diff
@@ -542,6 +542,7 @@ func (h *TunnelHandler) ServeStream(stream *h2mux.MuxedStream) error {
 	h.logRequest(req, cfRay, lbProbe)
 	if websocket.IsWebSocketUpgrade(req) {
 		conn, response, err := websocket.ClientConnect(req, h.tlsConfig)
+		h.logger.WithFields(log.Fields{"connectionID": h.connectionID, "status": response.StatusCode}).Info("incrementResponses")
```

Comment:
Response is logged in line 555 and line 568.

Reply:
+1

Reply:
Although I see that errors are logged differently, without the response.  And also assumes that all errors are status code 502.  I think it makes more sense to log the response in all cases.  I agree that the logging is redundant